lib/wificaptive/src/WifiCaptive.h

```diff
@@ -36,62 +34,67 @@
 #define WIFI_LAST_USED_SSID_KEY "wifi_ssid"
 #define WIFI_LAST_USED_PSWD_KEY "wifi_pswd"
 
-class WifiCaptive {
-    private:
-        struct WifiCredentials {
-	        String ssid;
-	        String pswd;
-        };
-        struct Network {
-		    String ssid;
-		    int32_t rssi;
-		    bool open;
-            bool saved;
-	    };
-        
-        DNSServer* _dnsServer;
-        AsyncWebServer* _server;
-        String _ssid = "";
-        String _password = "";
-
-        std::function<void()> _resetcallback;
-
-        WifiCredentials _savedWifis[WIFI_MAX_SAVED_CREDS];
-        WifiCredentials _lastUsed;
-
-        void setUpDNSServer(DNSServer &dnsServer, const IPAddress &localIP);
-        void setUpWebserver(AsyncWebServer &server, const IPAddress &localIP);
-        uint8_t connect(String ssid, String pass);
-        uint8_t waitForConnectResult(uint32_t timeout);
-        uint8_t waitForConnectResult();
-        void readWifiCredentials();
-        void saveWifiCredentials(String ssid, String pass);
-        void saveLastUsed(String ssid, String pass);
-        std::vector<WifiCredentials> matchNetworks(std::vector<Network> &scanResults, WifiCaptive::WifiCredentials wifiCredentials[]);
-        std::vector<Network> getScannedUniqueNetworks(bool runScan);
-        std::vector<Network> combineNetworks(std::vector<Network> &scanResults, WifiCaptive::WifiCredentials wifiCredentials[]);
-
-    public:
-        /// @brief Starts WiFi configuration portal.
-        /// @return True if successfully connected to provided SSID, false otherwise.
-        bool startPortal();
-
-        /// @brief Checks if any ssid is saved
-        /// @return True if any ssis is saved, false otherwise
-        bool isSaved();
-
-        /// @brief Resets all saved credentials
-        void resetSettings();
-
-        /// @brief sets the function callback that is triggered when uses performs soft reset
-        /// @param func reset callback
-        void setResetSettingsCallback(std::function<void()> func);
-
-        /// @brief Connects to the saved SSID with the best signal strength
-        /// @return True if successfully connected to saved SSID, false otherwise.
-        bool autoConnect();
+class WifiCaptive
+{
+private:
+    struct WifiCredentials
+    {
+        String ssid;
+        String pswd;
+    };
+    struct Network
+    {
+        String ssid;
+        int32_t rssi;
+        bool open;
+        bool saved;
+    };
+
+    DNSServer *_dnsServer;
+    AsyncWebServer *_server;
+    String _ssid = "";
+    String _password = "";
+    String _api_server = "https://trmnl.app";
```

Comment:
same comment as before, ideally point to our constant default

Reply:
done!